### PR TITLE
feature: added landing page

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added landing page
Preview Link: https://deploy-preview-1--gallant-pasteur-062544.netlify.app/

Screenshot (31)

### CODE REVIEW

**Comment by sharathnair9999** [comment]
Looks clean. You can merge

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+body {
+  width: 100%;
+}
+.main-body {
+  width: 100%;
+}
+.trivia-cards-container {
+  display: flex;
+  flex-direction: row;
```
Comment (GuptaEesh) [inline css/main.css]: you don't need to specify this. As the default direction is row.

**Review by GuptaEesh — COMMENTED** [review]
Clean code. Added a small suggestion. Other than that, good to go.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  border-radius: 1rem;
+  cursor: pointer;
+}
+.card-content-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 100%;
+  padding: 1rem 1rem 1rem 1rem;
```
Comment (Ronak-Mutha) [inline css/main.css]: If values are same, you can shorten it to 1 value. i.e. `padding: 1rem;`

**Review by Ronak-Mutha — COMMENTED** [review]
Added a suggestion, rest code looks clean and good to merge.

File: css/main.css (hunk 2 above)
Comment (omgaikwad) [reply css/main.css]: Sure!

File: css/main.css (hunk 1 above)
Comment (omgaikwad) [reply css/main.css]: Sure

**Pull request merged**